rma_account/models/rma_order_line.py

```diff
@@ -82,7 +82,6 @@ def _compute_refund_count(self):
         string="Refund Lines",
         domain=[
             ("move_id.move_type", "in", ["in_refund", "out_refund"]),
-            ("exclude_from_invoice_tab", "=", False),
```

Comment:
Replace by `("display_type", "=", "product"),` to keep the same behavior ?

Reply:
Done!